applications.json

```diff
@@ -7498,6 +7498,23 @@
                 "cpp",
                 "objective_c"
             ]
+        },
+        {
+            "short_description": "Safari extension to save your working sessions",
+            "categories": [
+                "productivity",
+                "extensions"
+            ],
+            "repo_url": "https://github.com/AlexPerathoner/Sessions",
+            "title": "Sessions",
+            "icon_url": "https://github.com/AlexPerathoner/Sessions/blob/master/Sessions/Assets.xcassets/AppIcon.appiconset/512-1.png",
```

Comment:
It should be a link to an image.

Reply:
Please change to https://raw.githubusercontent.com/AlexPerathoner/Sessions/master/Sessions/Assets.xcassets/AppIcon.appiconset/512-1.png